src/components/Navbar/Navbar.tsx

```diff
@@ -102,6 +102,11 @@ const Navbar = ({
         return `${profile.given_name} ${profile.family_name}`;
     }
 
+    const hasTranslations = (languages: Languages | undefined): boolean => {
+        if (languages) return Object.keys(languages).length > 0;
+        return false;
```

Comment:
Could we do it like this?
```js
return languages && Object.keys(languages).length > 0;
```

Reply:
Much cleaner! Had to cast languages as a boolean as it is set to possibly be undefined though.